InternetHospital.WebApi/Startup.cs

```diff
@@ -122,7 +122,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
             }
 
             app.UseStaticFiles();
-            app.UseMiddleware(typeof(ErrorHandlingMiddleware));
+            //app.UseMiddleware(typeof(ErrorHandlingMiddleware));
```

Comment:
Why did you comment Sasha`s middleware?

Reply:
Because there is no middleware, only template. I think we will have to create this class in future.